_docs_integrate/data-model-overview.md

```diff
@@ -173,29 +184,29 @@ A File further has its content, of course. But since this is not a JSON property
 
 ## IdentityDeletionProcess
 
-If you want to delete your [Identity](#identity), the actual deletion of all data associated with that Identity will only take place after a certain grace period has ended.
+If you want to [delete your Identity]({% link _docs_integrate/delete-identities.md %}), the actual deletion of all data associated with that [Identity](#identity) will only take place after a certain grace period has ended.
 Up until then, the data about the deletion process is stored in an object called IdentityDeletionProcess.
 It is possible to [cancel an IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}) that hasn't reached the end of its grace period, yet.
-Furthermore, the deletion of an Identity can be initiated by the Backbone Admin UI.
+Furthermore, the deletion of an Identity can be triggered via the Backbone Admin UI.
 In this case, the Identity has to [approve the IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-approve-identitydeletionprocess.md %}) in order to start the grace period.
-The IdentityDeletionProcess can also be [rejected]({% link _docs_use-cases/use-case-transport-reject-identitydeletionprocess.md %}) if an Identity does not agree with its deletion initiated by the Backbone Admin UI.
+The IdentityDeletionProcess can also be [rejected]({% link _docs_use-cases/use-case-transport-reject-identitydeletionprocess.md %}) if an Identity does not agree with its deletion triggered via the Backbone Admin UI.
```

Comment:
I intentionally use the wording "initiating" when the "InitiateIdentityDeletionProcessUseCase" is called and "triggering" when the Identity deletion is triggered via the Backbone Admin UI to differentiate between these two possibilities. This is particularly useful because both options lead to a different starting situation: In the first case, an IdentityDeletionProcess with the status `"Approved"` is created, while in the second case only an IdentityDeletionProcess with the status `"WaitingForApproval"` is created.

Due to a review comment, I have also changed the wording "**by** Backbone Admin UI" to "**via** Backbone Admin UI" so that it does not seems as if the Backbone Admin UI is an acting entity, but is more clearly recognizable as a tool used by a Backbone administrator.